app/main/forms.py

```diff
@@ -0,0 +1,45 @@
+from flask.ext.wtf import Form
+from wtforms.fields import StringField, SubmitField, IntegerField, \
+    FloatField, DateField
+from wtforms import validators
+from ..models import Agency
+from wtforms.ext.sqlalchemy.fields import QuerySelectField
+from wtforms.validators import DataRequired
+from .. import db
+import datetime as datetime
+
+
+class NewIncidentForm(Form):
+    license_plate_number = StringField('License Plate Number', [
```

Comment:
Yeah, I think it was Vehicle ID that was required and License Plate Number was optional. 

Fields: 
- Agency 
- Vehicle ID 
- License Plate Number
- Description (ex. Driver was sleeping)
- Idling Duration (minutes) 
- Location (people will put down a pin)
- Photo